docs/transpile/qiskit-transpiler-service.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Transpile circuits remotely with the Qiskit transpiler service
+description: What is the Qiskit transpiler service and how to use it
+
+---
+# Transpile circuits remotely with the Qiskit transpiler service
+
+The Qiskit transpiler service provides transpilation capabilities on the cloud. In addition to the traditional [Qiskit transpiling capabilities](index), you can use IBM Quantum Cloud resources for transpilation tasks while taking advantage of AI-powered transpiler passes.
+
+The Qiskit transpiler service offers a Python library to seamlessly integrate this service and its capabilities into your current Qiskit patterns and workflows.
+
+<Admonition type="note">
+    This experimental service is only available for IBM Quantum Premium Plan users.
+    The service is an alpha release, subject to change.
+</Admonition>
+
+<span id="install-transpiler-service"></span>
+## Install the qiskit-transpiler-service package
+
+To use the Qiskit transpiler service, install the `qiskit-transpiler-service` package:
+
+```sh
+pip install qiskit-transpiler-service
+```
+
+By default, the package will try to authenticate to IBM Quantum services with the defined Qiskit API token, and will take your token from the `QISKIT_IBM_TOKEN` environment variable or from the file `~/.qiskit/qiskit-ibm.json` (under the section `default-ibm-quantum`).
+
+## qiskit-transpiler-service transpile options
+
+- `target` (optional, str) - Refers to a backend name as it would be expected by QiskitRuntimeService (i.e., `ibm_sherbrooke`). If this is set, the transpile method uses the layout from the specified backend for the transpilation operation. If any other option is set that impacts these settings, such as `coupling_map`, the `target` settings will be overridden.
```

Comment:
```suggestion
- `target` (optional, str) - A backend name as it would be expected by QiskitRuntimeService (for example, `ibm_sherbrooke`). If this is set, the transpile method uses the layout from the specified backend for the transpilation operation. If any other option is set that impacts these settings, such as `coupling_map`, the `target` settings are overridden.
```